cfme/tests/networks/test_sdn_addresses.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+
+from cfme import test_requirements
+from cfme.cloud.provider.openstack import OpenStackProvider
+from cfme.networks.views import FloatingIpView
+
+pytestmark = [
+    test_requirements.sdn,
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.provider([OpenStackProvider], scope="module")
+]
+
+
+@pytest.fixture()
+def network_manager(appliance, provider):
+    network_managers = appliance.collections.network_providers.filter({"provider": provider})
+    yield network_managers.all()[0]
```

Comment:
This is bit suspicious to me. How is it ensured the first returned network_manager is the one the test wants?

Reply:
Well, how can openstack provider have multiple network managers? Never seen this option in CFME.
Also we are using the same fixture for storage managers and it works well. : https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/storage/test_manager.py#L31